src/main/resources/assets/materialisation/lang/fr_fr.json

```diff
@@ -0,0 +1,120 @@
+    "item.materialisation.axe_head_pattern":"Modèle de tête de pioche",
+    "item.materialisation.shovel_head_pattern":"Modèle de tête de pelle",
+    "item.materialisation.pickaxe_head_pattern":"Modèle de tête de pioche",
+    "item.materialisation.megaaxe_head_pattern":"Modèle de tête de méga hache",
+    "item.materialisation.sword_blade_pattern":"Modèle de lame d'épée",
+    "item.materialisation.hammer_head_pattern":"Modèle de tête de marteau",
+    "container.materialisation.materialising_table":"Table de matérialisation",
+    "container.materialisation.material_preparer":"Préparateur de matériaux",
+    "text.materialisation.broken":"Durabilité actuelle: §cCassé",
+    "text.materialisation.max_durability":"Durabilité max : %d",
+    "text.materialisation.durability":"Durabilité actuelle : %s§f (%s§f%%)",
+    "text.materialisation.breaking_speed":"Vitesse de minage : %.2f",
+    "text.materialisation.mining_level":"Niveau minier : %d",
+    "text.materialisation.tool_handle_durability_multiplier":"Multiplicateur de durabilité : %s",
+    "text.materialisation.tool_handle_speed_multiplier":"Multiplicateur de vitesse de minage : %s",
+    "text.materialisation.head_part_speed":"Vitesse de minage : %s",
+    "text.materialisation.head_part_durability":"Durabilité de la pièce : %s",
+    "text.materialisation.head_part_damage":"Points d'attaque : %s",
+    "material.materialisation.minecraft.wood":"bois",
+    "material.materialisation.minecraft.stone":"pierre",
+    "material.materialisation.minecraft.iron":"fer",
+    "material.materialisation.minecraft.gold":"or",
+    "material.materialisation.cotton-resources.copper":"cuivre",
+    "material.materialisation.cotton-resources.silver":"argent",
+    "material.materialisation.cotton-resources.lead":"plomb",
+    "material.materialisation.cotton-resources.zinc":"zinc",
+    "material.materialisation.cotton-resources.aluminum":"aluminium",
+    "material.materialisation.cotton-resources.cobalt":"cobalt",
+    "material.materialisation.cotton-resources.tin":"étain",
+    "material.materialisation.cotton-resources.titanium":"titane",
+    "material.materialisation.cotton-resources.tungsten":"tungsten",
+    "material.materialisation.cotton-resources.platinum":"platine",
+    "material.materialisation.cotton-resources.palladium":"palladium",
+    "material.materialisation.cotton-resources.osmium":"osmium",
+    "material.materialisation.cotton-resources.iridium":"iridium",
+    "material.materialisation.cotton-resources.steel":"acier",
+    "material.materialisation.cotton-resources.brass":"laiton",
+    "material.materialisation.cotton-resources.electrum":"électrum",
+    "material.materialisation.cotton-resources.mercury":"mercure",
+    "material.materialisation.netherthings.nether":"brique du Nether",
+    "material.materialisation.netherthings.glowstone":"pierre lumineuse",
+    "material.materialisation.netherthings.vibranium":"vibranium",
```

Comment:
Thank you for helping! But how about changing the casing to match the English localisation? Thank you!

Reply:
I would argue that the casing is part of the localization. See the name for "Wooden Pickaxe":
![image](https://user-images.githubusercontent.com/25536265/66213021-3e4ebb80-e6bf-11e9-8a43-5017beef9a61.png)
![image](https://user-images.githubusercontent.com/25536265/66213046-47d82380-e6bf-11e9-85c9-786697b9ff4e.png)
The same material names do show up in the config menu, which may be a bit more troubling, but I do not think it looks weird in French.
![image](https://user-images.githubusercontent.com/25536265/66213112-68a07900-e6bf-11e9-8aef-fb5ae97db5c7.png)

Reply:
Oh right those materials are moved to the back, I will merge this thanks!